modules/gapi/test/gapi_graph_meta_tests.cpp

```diff
@@ -99,7 +99,12 @@ TEST(GraphMeta, Streaming_AccessInput) {
     cv::GComputation graph(cv::GIn(in), cv::GOut(out1, out2));
 
     auto ccomp = graph.compileStreaming();
-    ccomp.setSource<cv::gapi::wip::GCaptureSource>(findDataFile("cv/video/768x576.avi", false));
+    const auto path = findDataFile("cv/video/768x576.avi", false);
```

Comment:
> false

Why is `false` here?
This file is part of `opencv_extra` and its searching must be mandatory.

Reply:
It is author's idea. But without `false` tests are failed on local pc. With `false` are skipped.

Reply:
>  failed on local pc

`OPENCV_TEST_DATA_PATH` must be properly configured. See Wiki pages how to use OpenCV tests.

`false` must be used only for optional test data on external data shares.
Otherwise, related test data can be removed / moved / renamed in opencv_extra. And these tests would just skip.

Reply:
Ok. Removed `false` from `findDataFile`.